docs/development.rst

```diff
@@ -12,3 +12,11 @@ To run tests:
 .. code-block:: sh
 
     nosetests
+
+To make a release we use `zest.releaser`_.
+
+.. code-block:: sh
+
+    fullrelease
```

Comment:
In other projects, I have added a ``release:`` target in the ``Makefile``, which simply calls ``fullrelease``.

We could use this too here, which allows us to change the release tool if needed without touching the docs.